consensus/view_models/milestone.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_CONCENSUS_VIEW_MODELS_MILESTONE_H__
+#define __COMMON_CONCENSUS_VIEW_MODELS_MILESTONE_H__
+
+#include "common/errors.h"
+#include "common/model/transaction.h"
+#include "consensus/tangle/tangle.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef trit_array_p milestone_view_t;
```

Comment:
what's a `milestone_view`?

Reply:
Not so sure about it, maybe we will have to have another table for representing milestones, that are transactions with index field, or rather have an optional field in `iota_transaction` table for index..